service-loadbalancer/service_loadbalancer.go

```diff
@@ -430,6 +440,17 @@ func dryRun(lbc *loadBalancerController) {
 	}
 }
 
+// getErrorIndex returns the default page to return for traffic
+// that reach the load balancer and does not match any acl
+func getErrorIndex() []byte {
```

Comment:
I you accept my previous suggestion of making this a type, this function will move in there